index.bs

```diff
@@ -11285,45 +11317,55 @@ being thrown.  When not in strict mode, the assignment attempt will be ignored.
 
 <h4 id="es-operations">Operations</h4>
 
-For each unique [=identifier=]
-of an [=exposed=] [=operation=]
-defined on the [=interface=], there
-must exist a corresponding property,
-unless the [=effective overload set=]
-for that [=identifier=] and [=operation=]
-and with an argument count of 0 has no entries.
-
-The characteristics of this property are as follows:
-
-*   The name of the property is the [=identifier=].
-*   The location of the property is determined as follows:
-    *   If the operation is [=static operations|static=], then
-        the property exists on the [=interface object=].
-    *   Otherwise, if the operation is [=unforgeable=] on
-        the interface or if the interface was declared with the [{{Global}}] [=extended attribute=],
-        then the property exists on every object that implements the interface.
-    *   Otherwise, the property exists solely on the interface’s [=interface prototype object=].
-*   The property has attributes
-    { \[[Writable]]: |B|, \[[Enumerable]]: <emu-val>true</emu-val>, \[[Configurable]]: |B| },
-    where |B| is <emu-val>false</emu-val> if the operation is
-    [=unforgeable=] on the interface,
-    and <emu-val>true</emu-val> otherwise.
-*   The value of the property is the result of [=creating an operation function=] given the operation,
-    the interface, and the [=relevant Realm=] of the object that is the location of the property.
+For each unique [=identifier=] of an [=exposed=] [=operation=] defined on the [=interface=],
+there must exist a corresponding property.
```

Comment:
Here you kept the must, although eventually it's unnecessary (and you've been deleting it from previous sections, making these intros just statements of fact---which is good). Probably best to delete it from here too, although you'll need to---like the above---add some temporary musts for unforgeable and globals.

Reply:
Partially addressed in c6eb400